homepage.php

```diff
@@ -1,73 +1,112 @@
+				}
+		else{
+			$User = $_SESSION['login_user'];
+		}
+?>
+
 <!DOCTYPE html>
 <html lang = "en">
 	<head>
-		<title> Homepage</title>
+		<title>Homepage</title>
 		<meta chrset="utf-8"/>
 		<meta name="Keywords" content = "html5"/>
 		<meta name = "Author" content = "Tony Nguyen"/>
 		<meta name = "Description" content = "Music database"/>
 		<meta name = "viewport" content = "width=device-width. initial-scale=1"/>
 		<!--Connected to css -->
-		<link rel ="stylesheet" href = "homepage.css"> 
+		<link rel ="stylesheet" href = "homepage.css">
 		
 	</head>
 <body>
+	<!--Set for grid system for the page -->
 	<div class ="grid-container">
+		<!-- The div for nav and in the div it contain the logo and a function to called the nav -->
 		<div class="navigation">
+			<!-- Set a div that contain logo for the music database -->
 			<div class="logo">
 				<a href = "homepage.php"><img src = "images/logo8.png" height = "100"></a>
 			</div>
-			<nav>
-				<ul>
-					<li><a href = "homepage.php">Home</a></li>
-					<li><a href = "musicplayer.php">Music Play</a>
-						<ul>
-							<li><a href= "Query1.php">Query1</a></li>
-							<li><a href= "Query2.php">Query2</a></li>
-						</ul>
-					</li>
-					<li><a href = "login.php">Contact</a></li>
-					<li><a href = "login.php">LoginOut</a></li>
-					<li><a href = "signup.php">SignUp</a></li>
-				</ul>
-			</nav>
+			<?php
+				//Pulls the links from the nav_normal.php page and places them in the navigation div
+				require 'nav_normal.php'; //'require' is 100% needed for the site to run 
+			?>
 		</div>
+		<!-- Div that holds the banner -->
 		<div class="img_1">
-			<img src = "images/img.png">
+			<img src = "images/banner.jpg">
 		</div>
 		<div class="content_1"><!-- Holds the main page content -->
 			<h1>Welcome to MUSIC MP3</h1>
 			<p>MUSIC MP3 is a free music player application. With a high-quality music store, you will have a great music experience.</p>
-			<p>Digital music service with millions of high quality copyrighted songs , helping you listen to music, download music, upload and sync my music store on multiple devices.</p>
+			<p >Digital music service with millions of high quality copyrighted songs , helping you listen to music, download music, upload and sync my music store on multiple devices.</p>
 		</div>
+		<!-- Holds the Imges which all be the link to other special playlist-->
 		<div class="content_4">
 			<Space></Space>
 			<div class="images1">
-				<a href = "#"><img src = "images/img.png"></a>
+				<a href = "Query8.php"><img src = "images/indie.jpg" class="image1"></a>
+				<!--Div for overlay techniques so that when the user hover it hide the images and display text -->
+				<div class="overlay">
+    				<div class="text"><a href = "Query8.php">Indie</a></div>
+  				</div>
 			</div>
 			<div class="images2">
-				<a href = "#"><img src = "images/img.png"></a>
+				<a href = "Query7.php"><img src = "images/R&B.jpg" class="image2"></a>
```

Comment:
26/8/2021: Adding a img that this use to reparent the idea a music type. When the user hover on the img when the overlay techniques will work and it will show the link to the query that store a table of all the music in that specific type of music. Eg. If the img indicate to rock music then when the user hover on that image then it will show up will the text rock and when the user link on the link it will deliver the user to a page of all rock music.